sawmill-benchmark/pom.xml

```diff
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>io.logz.sawmill</groupId>
             <artifactId>sawmill-core</artifactId>
-            <version>${project.version}</version>
+            <version>${revision}</version>
```

Comment:
This reminded me - do we have a plan to automate the benchmark build?

Reply:
no, but it needs version